### ISSUE TITLE
There is not 'dbTables' in 'doc' folder

### ISSUE THREAD

**Issue by AlexRITIAN** [quartz-scheduler/quartz#687]
I download the 'quartz-2.3.0-distribution.tar.gz' from 'http://www.quartz-scheduler.org/downloads/'.
but i can not find the table-creation SQL scripts in the “docs/dbTables” directory.
Has the script location changed? Or is the script no longer provided?

**Comment by sahil-ag** [comment]
I think they were moved to `src\org\quartz\impl\jdbcjobstore`

**Comment by jhouserizer** [comment]
But they should be in the doc folder (or somewhere easy to find) in the main distribution build.

**Comment by debashisdeb** [comment]
Agreed, the location has been changed to a non trivial path : `src\org\quartz\impl\jdbcjobstore`
